src/main/java/org/sqlite/JDBC.java

```diff
@@ -49,7 +49,7 @@ public boolean jdbcCompliant() {
         return false;
     }
 
-    public Logger getParentLogger() throws SQLFeatureNotSupportedException {
+    public java.util.logging.Logger getParentLogger() throws SQLFeatureNotSupportedException {
```

Comment:
this needs to return a `java.util.logging.Logger` as per https://docs.oracle.com/javase/8/docs/api/java/sql/Driver.html#getParentLogger--

Reply:
It is returning that. I added the package because the new sqlite-jdbc Logger class conflicts.

Reply:
right, i misread the diff somehow, please ignore